.github/workflows/deploy.yml

```diff
@@ -272,11 +272,22 @@ jobs:
         env:
           COMMITTER_TOKEN: ${{ secrets.HOMEBREW_GITHUB_API_TOKEN }}
 
-  publish_docs:
-    name: Publish docs to Netlify
+  merge_crowdin_pr:
+    name: Merge Crowdin PR
     runs-on: ubuntu-latest
     needs: release_please
     if: ${{ needs.release_please.outputs.release_created == 'true' }}
+    continue-on-error: true
+    steps:
+      - name: Merge | Merge Crowdin PR
+        run: gh pr merge --squash i18n_master
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+
+  publish_docs:
+    name: Publish docs to Netlify
+    runs-on: ubuntu-latest
+    needs: merge_crowdin_pr
     steps:
       - name: Setup | Checkout
         uses: actions/checkout@v3
```

Comment:
I think this needs `with: ref: master` to include the just-merged commit.

Reply:
Yep! Excellent point! 👍

Reply:
And I just noticed `notarize_and_pkgbuild` will also build the docs. Maybe all deploy workflows should use it?

Reply:
I've gone ahead and added the ref and dependency on the `merge_crowdin_pr` step. I think that should be fine for now 👍